src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-v1_2.xml

```diff
@@ -29,6 +29,9 @@
                     <key-store-clear-password password="password"/>
                 </key-store-ssl-certificate>
             </ssl-context>
+            <ssl-context name="client-ssl-context-key-manager">
+                <key-manager algorithm="SunX509"/>
```

Comment:
* SunX509 wont work on IBM JDK. Does PKIX work with key-manager?
* is it possible reference (test) keystore, e.g. client-keystore ?